asterisk/build-asterisk.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+PROGNAME=$(basename $0)
+
+if test -z ${ASTERISK_VERSION}; then
+    echo "${PROGNAME}: ASTERISK_VERSION required" >&2
+    exit 1
+fi
+
+set -ex
+
+mkdir /usr/src/asterisk
```

Comment:
If you got inspiration from that github project, please mention it here as a comment

Reply:
done.